src/Infrastructure/LeanCode.Serialization/JsonConverters/JsonLaxDateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,29 @@
+using System.Buffers;
+using System.Globalization;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace LeanCode.Serialization;
+
+public class JsonLaxDateTimeOffsetConverter : JsonConverter<DateTimeOffset>
+{
+    private static readonly UTF8Encoding encoder = new(false);
+    private readonly string format = "yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffZ";
```

Comment:
Is that `Z` a custom format specifier or a literal? Because neither seems correct for me here (but I don't exactly remember our requirements on how the offset can be specified).

Please make sure that a string like `2023-02-06T21:37:42.666+01:00` can be both written and read correctly.